tests/e2e/runner-pod/01-assert.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 60
+commands:
+  - script: kubectl get pods -n "$NAMESPACE" --no-headers=true -l vm.neon.tech/name=example | [ $(wc -l) -eq 1 ]
```

Comment:
This _may_ fail due to the PRESERVE_RUNNER_PODS env var in CI (which in turn sets the `nodelete` build tag) — basically, in order to keep VM runner pod logs available in CI, we don't delete the runner pod.

Reply:
I see, but still there should be only one running pod, right? I can query only the pods that are running.

Reply:
For now, yes. With container-mgr enabled, no: the original pod will remain "running" because even though the neonvm-runner container will be killed, the the container-mgr container will still be running.

Reply:
I thought with crictl PR merged container-mgr will also be killed when runner dies, no?

Reply:
Basically with crictl enabled, we would have to make a choice between deleting the pod (to kill container-mgr) or leaving it running. In CI, we want to keep the runner pods around to be able to fetch their logs, which means we can't delete the pod (and therefore can't kill container-mgr).

Reply:
Would killing container-mgr (and not the whole pod) be a soultion. Thus pod is preserved when we have `nodelete`?

Reply:
I checked this when working on the original PR and AFAICT there isn't a clean way to kill a container within a pod — but, I'd love to find a solution :sweat_smile:

Reply:
I made it work with deleting the entire runner pod (kubectl delete). Works with both crictl and `--skip-delete` enabled.

Reply:
For deleting a specific container container in a pod, you can do this: `kubectl exec <PodName> --container <ContainerName> -- kill 1`.